### PR TITLE
Add canceled refund status

### PR DESCRIPTION
**Pull request by dabroek**
Add canceled status to list of refund statuses

### CODE REVIEW

File: source/payments/refunds.rst [hunk 1]
```diff
@@ -32,7 +32,7 @@ Refunds have their own status, independent of the Payment or Order they were cre
   The refund is ready to be sent to the bank. You can still cancel the refund if you like.
 
 *processing*
-  The refund is being processed. Cancellation is no longer possible.
+  The refund is being processed. Cancelation is no longer possible.
 
 *refunded*
   The refund has been settled to your customer.
```
Comment (sandervanhooft) [inline source/payments/refunds.rst:38]: A bit in doubt here. Is this appropriate English?

`The refund has been settled to your customer.`

Comment (dabroek) [reply source/payments/refunds.rst:38]: Correct would be "The refund is settled with your customer."

More comprehensible could be: "The refund has been completed and your customer has received it."

Comment (dabroek) [reply source/payments/refunds.rst]: FYI. Added the disclaimer that "the funds can still be on their way."

Comment (sandervanhooft) [reply source/payments/refunds.rst]: Even better!

**Review by sandervanhooft — APPROVED** [review]

**Review by mollierobbert — APPROVED** [review]

**Pull request merged**